[2025-03-21T12:50:45.551+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: transactions_pipeline.create_fraud_table manual__2025-03-21T12:50:21.650471+00:00 [queued]>
[2025-03-21T12:50:45.567+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: transactions_pipeline.create_fraud_table manual__2025-03-21T12:50:21.650471+00:00 [queued]>
[2025-03-21T12:50:45.570+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2025-03-21T12:50:45.590+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): create_fraud_table> on 2025-03-21 12:50:21.650471+00:00
[2025-03-21T12:50:45.596+0000] {standard_task_runner.py:57} INFO - Started process 270 to run task
[2025-03-21T12:50:45.600+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'transactions_pipeline', 'create_fraud_table', 'manual__2025-03-21T12:50:21.650471+00:00', '--job-id', '531', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_pipeline.py', '--cfg-path', '/tmp/tmp78zn0_rz']
[2025-03-21T12:50:45.603+0000] {standard_task_runner.py:85} INFO - Job 531: Subtask create_fraud_table
[2025-03-21T12:50:45.668+0000] {task_command.py:415} INFO - Running <TaskInstance: transactions_pipeline.create_fraud_table manual__2025-03-21T12:50:21.650471+00:00 [running]> on host 1b7c3466167a
[2025-03-21T12:50:45.788+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='rabiizahnoune7@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='transactions_pipeline' AIRFLOW_CTX_TASK_ID='create_fraud_table' AIRFLOW_CTX_EXECUTION_DATE='2025-03-21T12:50:21.650471+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-21T12:50:21.650471+00:00'
[2025-03-21T12:50:45.792+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-21T12:50:45.794+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n        docker exec hive beeline -u "jdbc:hive2://localhost:10000" -e "\n        CREATE EXTERNAL TABLE IF NOT EXISTS fraud_detections (\n            transaction_id STRING,\n            date_time STRING,\n            amount DOUBLE,\n            customer_id STRING,\n            location STRING,\n            fraud_reason STRING\n        )\n        STORED AS PARQUET\n        LOCATION \'/data/fraud_detections\';"\n        ']
[2025-03-21T12:50:45.806+0000] {subprocess.py:86} INFO - Output:
[2025-03-21T12:50:46.951+0000] {subprocess.py:93} INFO - SLF4J: Class path contains multiple SLF4J bindings.
[2025-03-21T12:50:46.952+0000] {subprocess.py:93} INFO - SLF4J: Found binding in [jar:file:/opt/hive/lib/log4j-slf4j-impl-2.17.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
[2025-03-21T12:50:46.953+0000] {subprocess.py:93} INFO - SLF4J: Found binding in [jar:file:/opt/hadoop/share/hadoop/common/lib/slf4j-log4j12-1.7.25.jar!/org/slf4j/impl/StaticLoggerBinder.class]
[2025-03-21T12:50:46.954+0000] {subprocess.py:93} INFO - SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
[2025-03-21T12:50:47.113+0000] {subprocess.py:93} INFO - SLF4J: Actual binding is of type [org.apache.logging.slf4j.Log4jLoggerFactory]
[2025-03-21T12:50:48.344+0000] {subprocess.py:93} INFO - Connecting to jdbc:hive2://localhost:10000
[2025-03-21T12:50:48.734+0000] {subprocess.py:93} INFO - Connected to: Apache Hive (version 3.1.3)
[2025-03-21T12:50:48.735+0000] {subprocess.py:93} INFO - Driver: Hive JDBC (version 3.1.3)
[2025-03-21T12:50:48.736+0000] {subprocess.py:93} INFO - Transaction isolation: TRANSACTION_REPEATABLE_READ
[2025-03-21T12:50:48.882+0000] {subprocess.py:93} INFO - INFO  : Compiling command(queryId=root_20250321125048_ae5f4cc9-f9ad-43bf-bcc5-ff94c28dce04): CREATE EXTERNAL TABLE IF NOT EXISTS fraud_detections (
[2025-03-21T12:50:48.883+0000] {subprocess.py:93} INFO -             transaction_id STRING,
[2025-03-21T12:50:48.884+0000] {subprocess.py:93} INFO -             date_time STRING,
[2025-03-21T12:50:48.884+0000] {subprocess.py:93} INFO -             amount DOUBLE,
[2025-03-21T12:50:48.885+0000] {subprocess.py:93} INFO -             customer_id STRING,
[2025-03-21T12:50:48.886+0000] {subprocess.py:93} INFO -             location STRING,
[2025-03-21T12:50:48.886+0000] {subprocess.py:93} INFO -             fraud_reason STRING
[2025-03-21T12:50:48.887+0000] {subprocess.py:93} INFO -         )
[2025-03-21T12:50:48.887+0000] {subprocess.py:93} INFO -         STORED AS PARQUET
[2025-03-21T12:50:48.887+0000] {subprocess.py:93} INFO -         LOCATION '/data/fraud_detections'
[2025-03-21T12:50:48.888+0000] {subprocess.py:93} INFO - INFO  : Concurrency mode is disabled, not creating a lock manager
[2025-03-21T12:50:48.888+0000] {subprocess.py:93} INFO - INFO  : Semantic Analysis Completed (retrial = false)
[2025-03-21T12:50:48.888+0000] {subprocess.py:93} INFO - INFO  : Returning Hive schema: Schema(fieldSchemas:null, properties:null)
[2025-03-21T12:50:48.888+0000] {subprocess.py:93} INFO - INFO  : Completed compiling command(queryId=root_20250321125048_ae5f4cc9-f9ad-43bf-bcc5-ff94c28dce04); Time taken: 0.014 seconds
[2025-03-21T12:50:48.889+0000] {subprocess.py:93} INFO - INFO  : Concurrency mode is disabled, not creating a lock manager
[2025-03-21T12:50:48.889+0000] {subprocess.py:93} INFO - INFO  : Executing command(queryId=root_20250321125048_ae5f4cc9-f9ad-43bf-bcc5-ff94c28dce04): CREATE EXTERNAL TABLE IF NOT EXISTS fraud_detections (
[2025-03-21T12:50:48.889+0000] {subprocess.py:93} INFO -             transaction_id STRING,
[2025-03-21T12:50:48.890+0000] {subprocess.py:93} INFO -             date_time STRING,
[2025-03-21T12:50:48.890+0000] {subprocess.py:93} INFO -             amount DOUBLE,
[2025-03-21T12:50:48.891+0000] {subprocess.py:93} INFO -             customer_id STRING,
[2025-03-21T12:50:48.891+0000] {subprocess.py:93} INFO -             location STRING,
[2025-03-21T12:50:48.891+0000] {subprocess.py:93} INFO -             fraud_reason STRING
[2025-03-21T12:50:48.892+0000] {subprocess.py:93} INFO -         )
[2025-03-21T12:50:48.892+0000] {subprocess.py:93} INFO -         STORED AS PARQUET
[2025-03-21T12:50:48.893+0000] {subprocess.py:93} INFO -         LOCATION '/data/fraud_detections'
[2025-03-21T12:50:48.893+0000] {subprocess.py:93} INFO - INFO  : Starting task [Stage-0:DDL] in serial mode
[2025-03-21T12:50:48.893+0000] {subprocess.py:93} INFO - INFO  : Completed executing command(queryId=root_20250321125048_ae5f4cc9-f9ad-43bf-bcc5-ff94c28dce04); Time taken: 0.065 seconds
[2025-03-21T12:50:48.894+0000] {subprocess.py:93} INFO - INFO  : OK
[2025-03-21T12:50:48.894+0000] {subprocess.py:93} INFO - INFO  : Concurrency mode is disabled, not creating a lock manager
[2025-03-21T12:50:48.895+0000] {subprocess.py:93} INFO - No rows affected (0.131 seconds)
[2025-03-21T12:50:48.895+0000] {subprocess.py:93} INFO - Beeline version 3.1.3 by Apache Hive
[2025-03-21T12:50:48.900+0000] {subprocess.py:93} INFO - Closing: 0: jdbc:hive2://localhost:10000
[2025-03-21T12:50:48.942+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-21T12:50:48.971+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=transactions_pipeline, task_id=create_fraud_table, execution_date=20250321T125021, start_date=20250321T125045, end_date=20250321T125048
[2025-03-21T12:50:49.025+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-21T12:50:49.062+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
